src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,50 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    protected static final int MIN_PASSWORDS_LENGTH = 6;
+    protected static final int MIN_USERS_AGE = 18;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        if (user == null) {
+            throw new RuntimeException("User shouldn't be a null");
+        }
+
+        if (user.getLogin() == null) {
+            throw new NullPointerException("Users login shouldn't be a null");
```

Comment:
```suggestion
            throw new RuntimeException("Users login shouldn't be a null");
```
you should not throw NullPointerExcpetion
this exception is for java to throw, when there was operation on null